### ISSUE TITLE
Consider removing "BitOpNot" to avoid misusage.

### ISSUE THREAD

**Issue by stkao05** [Doist/bitmapist#14]
##### TLDR;
This is not really a "code bug", but rather a potential misusage issue with the "BitOpNot" operator. The "BitOpNot" does exactly what it supposes to do (which is flipping the bit). However, since we are using bitmapist as a stat tool, we expect it to give us the result of negation of a population set, which "BitOpNot" does not provide. 

##### Example 
Suppose you have total of 100 users. You use bitmapist to mark active user with event "active". Assume you have 5 active user and the bitmap data look like this: 
```
1011011
```

Now you want to count how many active user you have
```
print len(MonthEvents('active', now.year, now.month)) 
#print 5, correct number.
```

Now suppose you wanna know how many inactive user you have. 
```
print len(BitOpNot(MonthEvents('active', now.year, now.month)))  
#print 2. 
```

Here the negation of the "active user set" gives us size of 2 instead of 95. 

The fundamental problem is that the variable length bitmap data only contain information about "who are active", but it does not contain information about the population size.

**Comment by imankulov** [comment]
I agree that this can be misleading, but I don't think it's something we should remove. While `BitOpNot` by itself may sound useless, it's extremely useful in cases in combinations with other keys, something like

```python
active_users_without_featurex = MonthEvents('active') & ~MonthEvents('feature_x')
```
